_posts/2020-08-04-EOL-APIv1-APIv2.md

```diff
@@ -0,0 +1,17 @@
+---
+title: EOL Copr APIv1 and APIv2
+author: Miroslav Suchý
+layout: post
+---
+During Copr history, we got [three APIs](http://frostyx.cz/posts/copr-has-a-brand-new-api). For a long time, we maintained all versions.
+
+We decided that it is time to remove the old versions. We are going to start with APiv1.  
+
+The schedule is:
+
+  * September 2020 - print warning on STDERR when you use APIv1 from python3-copr.
```

Comment:
This is still not implemented :-(  I am not sure what is the proper way of implementing this (just doing `write(stderr, "something")` isn't a correct thing to do).  I'd like to see some historical examples in other libraries.